plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchSplitManager.java

```diff
@@ -56,6 +57,9 @@ public ConnectorSplitSource getSplits(
         if (tableHandle.getType().equals(QUERY)) {
             return new FixedSplitSource(ImmutableList.of(new ElasticsearchSplit(tableHandle.getIndex(), 0, Optional.empty())));
         }
+        else if (tableHandle.getType().equals(AGG)) {
```

Comment:
This can be folded into the previous branch:

```java
if (tableHandle.getType().equals(QUERY) || tableHandle.getType().equals(AGG)) {
    ...
}
```